pages/common/promtool.md

```diff
@@ -0,0 +1,20 @@
+# promtool
+
+> Tooling for the Prometheus monitoring system.
+> More information: <https://prometheus.io/docs/prometheus/latest/getting_started/>.
+
+- Check if the config files are valid or not (if present report errors):
+
+`promtool check config {{config_file.yml}}`
+
+- Check if the rule files are valid or not (if present report errors):
+
+`promtool check rules {{rules_file.yml}}`
+
+- Pass Prometheus metrics over stdin to check them for consistency and correctness:
```

Comment:
```suggestion
- Pass Prometheus metrics over `stdin` to check them for consistency and correctness:
```

Reply:
We shouldn't do this @navarroaxel&mdash;see https://github.com/tldr-pages/tldr/pull/5119#discussion_r558935557 and https://github.com/tldr-pages/tldr/commit/c0b847b4867a32020fb38380477086bca9b61aa8.

Reply:
@sbrl and @schneiderl changed their minds.

Reply:
Oh ok, cool.